scripts/build.ps1

```diff
@@ -604,8 +602,12 @@ function Get-ElapsedTime([System.Diagnostics.Stopwatch] $timer)
     return $timer.Elapsed
 }
 
-function Set-ScriptFailed
+function Set-ScriptFailed-OnError
```

Comment:
nit: function name doesn't match the Powershell naming convention.

Reply:
Renamed.